.gitmodules

```diff
@@ -27,3 +27,8 @@
 	path = third_party/opentelemetry-proto
 	url = https://github.com/open-telemetry/opentelemetry-proto
 	branch = master
+
+[submodule "third_party/nlohmann-json"]
+	path = third_party/nlohmann-json
+	url = https://github.com/nlohmann/json
+	branch = master
```

Comment:
Do we need specify the branch with tag like 3.9.1?

Reply:
Also do we need add this to include path to activate this submodule?

Reply:
> Also do we need add this to include path to activate this submodule?

Yes, and I will handle it separately.

Reply:
> Do we need specify the branch with tag like 3.9.1?

No, because `nlohmann-json` repo has two branches - `master` and `develop`. We point to `master` branch and the submodule actually points to release tag `v3.9.1`